data/campaigns/Eastern_Invasion/scenarios/12_Evacuation.cfg

```diff
@@ -1,166 +1,264 @@
 #textdomain wesnoth-ei
+
+#
+# if the player is already struggling, this scenario is easy. With no recall list, you can win by just rushing Gweddry/Owaec/Dacyn acrosss
+# and if you have a single castle of worthwhile recalls, it's still fairly easy to get across
+#
+# but a veteran player with a huge recall list will have difficulty getting everyone across the bridge safely, because of both the gold costs and the time limit
+# this is designed to equalize things a little bit, and make it a little harder for a player's recall list to snowball out of control
+#
+
+#
+# the extremely low turn limit is to discrouage winning via killing enemy leaders.
```

Comment:
Typo

Reply:
I saw this as well, but chose to ignore them in comments since a regular player won't see them. eg There's also an error with 'moreso' elsewhere.